test/e2e/template/helmfile/tmpl_test.go

```diff
@@ -6,8 +6,9 @@ import (
 	"testing"
 	"text/template"
 
-	"github.com/helmfile/helmfile/pkg/tmpl"
 	"github.com/stretchr/testify/require"
+
+	"github.com/helmfile/helmfile/pkg/tmpl"
```

Comment:
Out of curiosity, is there any way to auto-organize imports this way?

I thought the standard goimports doesn't support sectioning. I tend to use an internal tool for my non-opensource company projects. VS code seemed to group imports into two sections, one for standard libs and another for non-std libs but not exactly sure...